docker/compose/docker-compose-hq.yml

```diff
@@ -1,11 +1,12 @@
 web:
+  container_name: 'web'
```

Comment:
I don't think it is after all. I added it for Nginx to be able to link to the container, but since it's also added in dockerhq.sh under runserver-prod, it's redundant here.